frontend/src/store/configureStore.ts

```diff
@@ -21,7 +21,7 @@ import { errorReducer } from "../error/errorSlice";
 import { errorMiddleware } from "../error/errorMiddleware";
 import workspaceReducer  from "../modules/Workplace/redux";
 
-export const setupStore = (preloadedState?: any) =>
+export const setupStore = (preloadedState?: any) : any =>
```

Comment:
This change is making the check to fail. Adding `any` as the return type makes objects returned by the Redux store (state and actions) to be null and thus throwing compile errors. 

The return type is inferred an so there is no need to explicity set it here. Could you remove it?